src/rpc/rawtransaction.cpp

```diff
@@ -37,11 +38,11 @@
 
 #include <univalue.h>
 
-/** High fee for sendrawtransaction and testmempoolaccept.
- * By default, transaction with a fee higher than this will be rejected by the
- * RPCs. This can be overridden with the maxfeerate argument.
+/** High fee rate for sendrawtransaction and testmempoolaccept.
+ * By default, transaction with a fee rate higher than this will be rejected by
+ * the RPCs. This can be overridden with the maxfeerate argument.
  */
-constexpr static CAmount DEFAULT_MAX_RAW_TX_FEE{COIN / 10};
+const static CFeeRate DEFAULT_MAX_RAW_TX_FEE_RATE{COIN / 10};
```

Comment:
Style nit: The storage-class specifier is typically put at the start of the declaration in this project -- that is: `static const …` is typically used instead of `const static …` :-)

Reply:
Fixed.